third_party/2and3/requests/models.pyi

```diff
@@ -117,7 +117,7 @@ class Response:
     def __iter__(self) -> Iterator[bytes]: ...
     def __enter__(self) -> Response: ...
     def __exit__(self, *args: Any) -> None: ...
-    def next(self) -> PreparedRequest: ...
+    def next(self) -> None: ...
```

Comment:
This should be `Union[None, PreparedRequest]` to express it can be either. A nicer way to write unions with `None` is to use`Optional`, though: `Optional [PreparedRequest]`.

Reply:
Ah!  
TIL: `Optional[...]` is a shorthand notation for `Union[..., None]`. Thanks!